### ISSUE TITLE
Plugin uses deprecated method

### ISSUE THREAD

**Issue by NikolayMetchev** [autonomousapps/dependency-analysis-gradle-plugin#1334]
**Plugin version**
2.6.0

**Gradle version**
8.11.1

**JDK version**
17

**(Optional) Kotlin and Kotlin Gradle Plugin (KGP) version**
2.1.0

**Describe the bug**
Gradle is complaining about the use of deprecated method that will be removed in Gradle 9:
https://docs.gradle.org/8.11.1/userguide/upgrading_version_8.html#deprecate_get_dependency_project
It is used in GradleStrings.kt

**To Reproduce**
Use plugin with latest gradle

**Expected behavior**
no warnings

**Additional context**
```


Unknown:gradleStrings.kt |  
-- | --
at org.gradle.api.internal.artifacts.dependencies.DefaultProjectDependency.getDependencyProject(DefaultProjectDependency.java:82)at org.gradle.api.internal.artifacts.dependencies.DefaultProjectDependency_Decorated.getDependencyProject(Unknown Source)at org.gradle.api.internal.catalog.DelegatingProjectDependency.getDependencyProject(DelegatingProjectDependency.java:77)at com.autonomousapps.internal.utils.GradleStringsKt.toIdentifier(gradleStrings.kt:203)at com.autonomousapps.internal.utils.GradleStringsKt.toIdentifiers(gradleStrings.kt:180)at com.autonomousapps.tasks.FindDeclarationsTask$Companion$computeDeclarations$1.call(FindDeclarationsTask.kt:70)at com.autonomousapps.tasks.FindDeclarationsTask$Companion$computeDeclarations$1.call(FindDeclarationsTask.kt:65)at org.gradle.api.internal.provider.DefaultProvider.calculateOwnValue(DefaultProvider.java:72) |   |   | at org.gradle.api.internal.artifacts.dependencies.DefaultProjectDependency.getDependencyProject(DefaultProjectDependency.java:82) |   |   |   | at org.gradle.api.internal.artifacts.dependencies.DefaultProjectDependency_Decorated.getDependencyProject(Unknown Source) |   |   |   | at org.gradle.api.internal.catalog.DelegatingProjectDependency.getDependencyProject(DelegatingProjectDependency.java:77) |   |   |   | at com.autonomousapps.internal.utils.GradleStringsKt.toIdentifier(gradleStrings.kt:203) |   |   |   | at com.autonomousapps.internal.utils.GradleStringsKt.toIdentifiers(gradleStrings.kt:180) |   |   |   | at com.autonomousapps.tasks.FindDeclarationsTask$Companion$computeDeclarations$1.call(FindDeclarationsTask.kt:70) |   |   |   | at com.autonomousapps.tasks.FindDeclarationsTask$Companion$computeDeclarations$1.call(FindDeclarationsTask.kt:65) |   |   |   | at org.gradle.api.internal.provider.DefaultProvider.calculateOwnValue(DefaultProvider.java:72)
  |   | at org.gradle.api.internal.artifacts.dependencies.DefaultProjectDependency.getDependencyProject(DefaultProjectDependency.java:82) |  
  |   | at org.gradle.api.internal.artifacts.dependencies.DefaultProjectDependency_Decorated.getDependencyProject(Unknown Source) |  
  |   | at org.gradle.api.internal.catalog.DelegatingProjectDependency.getDependencyProject(DelegatingProjectDependency.java:77) |  
  |   | at com.autonomousapps.internal.utils.GradleStringsKt.toIdentifier(gradleStrings.kt:203) |  
  |   | at com.autonomousapps.internal.utils.GradleStringsKt.toIdentifiers(gradleStrings.kt:180) |  
  |   | at com.autonomousapps.tasks.FindDeclarationsTask$Companion$computeDeclarations$1.call(FindDeclarationsTask.kt:70) |  
  |   | at com.autonomousapps.tasks.FindDeclarationsTask$Companion$computeDeclarations$1.call(FindDeclarationsTask.kt:65) |  
  |   | at org.gradle.api.internal.provider.DefaultProvider.calculateOwnValue(DefaultProvider.java:72)


```

**Comment by autonomousapps** [comment]
Duplicate of https://github.com/autonomousapps/dependency-analysis-gradle-plugin/issues/1305
